UnityProject/Assets/Scripts/HealthV2/Living/Damage/BodyPartDamage.cs

```diff
@@ -544,26 +568,41 @@ public void ApplyTraumaDamage(float tramuaDamage, TramuticDamageTypes damageType
 			//We use dismember protection chance because it's the most logical value.
 			if(DMMath.Prob(SelfArmor.DismembermentProtectionChance * 100) == false)
 			{
-				if(damageType == TramuticDamageTypes.SLASH) { currentSlashCutDamage += tramuaDamage; }
-				if(damageType == TramuticDamageTypes.PIERCE) { currentPierceDamage += tramuaDamage; }
+				if (damageType == TramuticDamageTypes.SLASH) { currentSlashCutDamage += MultiplyTraumaDamage(tramuaDamage); }
```

Comment:
just making sure, this check will only work if ``damageType`` has SLASH flag and no other. Is this what you want? BTW make the body of the ``if`` have its own line.

Reply:
This only gets called with one damage type so there is no need to check for multiple flags. Slash is the default in case no argument was given.